.github/workflows/integration-test.yml

```diff
@@ -13,81 +13,83 @@ jobs:
-          cd elasticsearch-${{ steps.kbn_version.outputs.kbn_version }}/
-          bin/elasticsearch-plugin install -b https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-security/opendistro-security-${{ steps.opendistro_version.outputs.opendistro_version }}.zip
-          bash plugins/opendistro_security/tools/install_demo_configuration.sh -y -i
-          echo 'opendistro_security.unsupported.restapi.allow_securityconfig_modification: true' >> config/elasticsearch.yml
-          echo $(curl -s -o /dev/null -w ''%{http_code}'' https://localhost:9200 -u admin:admin -k)
-          bin/elasticsearch &
-          sleep 30
-          timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' https://localhost:9200 -u admin:admin -k)" != "200" ]]; do echo "ping ElasticSearch..."; sleep 5; done'
+          echo "::set-output name=opendistro_version::$(jq -r '.version' ./OpenSearch-Dashboards/plugins/security-dashboards-plugin/package.json | sed 's/[0-9]*$/0/')"
       
-      - name: Use kibana release commit
+      - name: Use OpenSearch Dashboards release commit
         run: |
-          cd ./kibana
-          git checkout tags/v${{ steps.kbn_version.outputs.kbn_version }} -b v${{ steps.kbn_version.outputs.kbn_version }}
+          cd ./OpenSearch-Dashboards
+          # git checkout tags/v${{ steps.osd_version.outputs.osd_version }} -b v${{ steps.osd_version.outputs.osd_version }}
       
       - name: Get node and yarn versions
         id: versions
         run: |
-          echo "::set-output name=node_version::$(cat ./kibana/.node-version)"
-          echo "::set-output name=yarn_version::$(jq -r '.engines.yarn' ./kibana/package.json)"
+          echo "::set-output name=node_version::$(cat ./OpenSearch-Dashboards/.node-version)"
+          echo "::set-output name=yarn_version::$(jq -r '.engines.yarn' ./OpenSearch-Dashboards/package.json)"
       
       - name: Setup node
         uses: actions/setup-node@v1
         with:
           node-version: ${{ steps.versions.outputs.node_version }}
           registry-url: 'https://registry.npmjs.org'
       
-      - name: Install correct yarn version for Kibana
+      - name: Install correct yarn version for OpenSearch Dashboards
         run: |
           npm uninstall -g yarn
           echo "Installing yarn ${{ steps.versions_step.outputs.yarn_version }}"
           npm i -g yarn@${{ steps.versions.outputs.yarn_version }}
       
-      - name: Bootstrap Kibana
+      - name: Check OpenSearch Running
+        continue-on-error: true
+        run: curl -XGET https://localhost:9200 -u 'admin:admin' -k
+
+      - name: Bootstrap OpenSearch Dashboards
         run: |
-          cd ./kibana
-          yarn kbn bootstrap --oss
+          cd ./OpenSearch-Dashboards
+          yarn osd bootstrap
       
-      - name: Run integration tests
+      - name: Run integration test
```

Comment:
If there are more than 1 tests, should we retain `tests`

Reply:
hmm..i thought I put tests here
fixed with latest push